scripts/SetupEnvAndRunPHPUnitTests.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+######################################################################
+# Variables
+######################################################################
+
+if [[ -z "$1" ]] || [[ -z "$2" ]] || [[ -z "$3" ]] || [[ -z "$4" ]] || [[ -z "$5" ]] || [[ -z "$6" ]] || [[ -z "$7" ]]
+then
+    echo "Not all required command line arguments were set. Please run the script again with the required arguments:
+        1: Email address associated with your SugarCRM Developer Builds Community account
+        2: Password associated with the above account
+        3: Sugar version (Example: 7.11)
+        4: Sugar edition (Options: Ult, Ent, Pro)
+        5: GitHub username that has access to sugarcrm/unit-tests
```

Comment:
This is a problem for me because I use 2FA. Github supports using SSH keys to authenticate. It would be good (and probably a better security practice) to have the option of using SSH keys.

Reply:
Good point!  I opened #51 to track this